docs/learn/fundamentals/transactions/unique-source-for-operation.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "When Operation Source Accounts differ from that of the Parent Transaction"
+sidebar_label: Unique Source Accounts for Operations
+description: "Learn"
+sidebar_position: 40
+---
+
+# When Operation Source Accounts differ from that of the Parent Transaction
+
+Let's review the unique case where an operation may have a source account different from that of the parent transaction envelope.
+
+There are 2 main situations where this may happen:
+
+1. Operations that affect more than one account
+2. Operations with a high threshold weight
```

Comment:
It would really be more like `Transactions that ...` and `Transactions with ...` here